src/gui/editors/PianoRoll.cpp

```diff
@@ -912,39 +912,71 @@ void PianoRoll::drawDetuningInfo( QPainter & _p, const Note * _n, int _x,
 		width() - m_whiteKeyWidth,
 		keyAreaBottom() - PR_TOP_MARGIN);
 
+	// Draws the lines for the detuning automation. Just as previously, we are still
```

Comment:
In this case I don't think the comment needs to reference the previous code. So this could be shortened a bit to something like "Draw lines for note detune, treating cubic-hermite as straight lines".